devise.rb

```diff
@@ -1,59 +1,52 @@
 run "if uname | grep -q 'Darwin'; then pgrep spring | xargs kill -9; fi"
 
-# GEMFILE
+# Gemfile
 ########################################
-inject_into_file 'Gemfile', before: 'group :development, :test do' do
+inject_into_file "Gemfile", before: "group :development, :test do" do
   <<~RUBY
-    gem 'devise'
-
-    gem 'autoprefixer-rails', '10.2.5'
-    gem 'font-awesome-sass'
-    gem 'simple_form', github: 'heartcombo/simple_form'
+    gem "autoprefixer-rails", "10.2.5"
+    gem "devise"
+    gem "font-awesome-sass"
+    gem "simple_form", github: "heartcombo/simple_form"
   RUBY
 end
 
-inject_into_file 'Gemfile', after: 'group :development, :test do' do
-  <<-RUBY
-  gem 'pry-byebug'
-  gem 'pry-rails'
-  gem 'dotenv-rails'
-  RUBY
+inject_into_file "Gemfile", after: 'gem "debug", platforms: %i[ mri mingw x64_mingw ]' do
+<<-RUBY
+
+  gem "dotenv-rails"
+  gem "pry-byebug"
+  gem "pry-rails"
```

Comment:
Do we need these two gems? Since the gem `debug` is installed by default, we can use [`binding.break`](https://github.com/ruby/debug#invoke-with-the-debugger) just like we would do with `binding.pry`.